src/Packetery/Module/Checkout.php

```diff
@@ -1013,4 +1015,15 @@ private function getTaxClassWithMaxRate() {
 
 		return $resultTaxClass;
 	}
+
+	/**
+	 * Tells if CURCY plugin is active.
+	 *
+	 * Support for CURCY - Multi Currency for Woocommerce plugin.
+	 *
+	 * @return bool
+	 */
+	public function isCurcyPluginActive() {
+		return is_plugin_active( self::CURCY_PLUGIN );
```

Comment:
Jaký smysl má zavedení takové jednořádkové metody, která se používá jen jednou?

Reply:
ta metoda je spis "placeholder" pro ty komentare, nez abych je strkal do mist vlastniho volani

Reply:
Metoda mi přijde, vzhledem k tomu, že název metody kopíruje samotné volání a komentář to jen rozvádí, jako zcela zbytečná. Takže pokud jde o mě, prosím odstranit.